### PR TITLE
application: syn flux pkg with metric

### PR DESCRIPTION
**Pull request by Xieql**
**What type of PR is this?**

/kind api-change
/kind bug
/kind documentation

**What this PR does / why we need it**:

The current application imports a package that is not consistent with the metric, which causes crashes.
This PR synchronizes the Flux package with the metric.

In addition, since the application uses Kustomization in Flux, this PR modifies the installation documentation of Flux.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cmd/fleet-manager/scheme/flux.go
Comment (hzxuzhonghu) [inline cmd/fleet-manager/scheme/flux.go:21]: kustomizev1/kustomizev1beta2

File: cmd/fleet-manager/scheme/flux.go
Comment (hzxuzhonghu) [inline cmd/fleet-manager/scheme/flux.go:23]: sourcev1beta2

File: pkg/apis/apps/v1alpha1/types.go
Comment (hzxuzhonghu) [inline pkg/apis/apps/v1alpha1/types.go:97]: Why cannot we use V1.GitRepo

File: cmd/fleet-manager/scheme/flux.go
Comment (hzxuzhonghu) [reply cmd/fleet-manager/scheme/flux.go:21]: ping for it @Xieql

File: pkg/apis/apps/v1alpha1/types.go
Comment (Xieql) [reply pkg/apis/apps/v1alpha1/types.go:97]: v1 is the latest version. we are currently using v1beta2.

Comment (zirain) [reply pkg/apis/apps/v1alpha1/types.go:97]: we're using [helm](https://github.com/fluxcd-community/helm-charts/releases/tag/flux2-2.7.0) to install flux, currently the helm chart does not support v1

Comment (zirain) [reply pkg/apis/apps/v1alpha1/types.go:97]: we need wait for next release with this PR: https://github.com/fluxcd-community/helm-charts/pull/176

File: pkg/fleet-manager/application_controller.go
Comment (hzxuzhonghu) [inline pkg/fleet-manager/application_controller.go:25]: nit: sourcev1beta2

**Review by hzxuzhonghu — APPROVED** [review]

**Comment by zirain** [comment]
please merge commit

**Comment by Xieql** [comment]
> 

may be we need a /lgtm label?

**Review by zirain — APPROVED** [review]
/lgtm

**Pull request merged**
